### DISCUSSION TITLE
Schema parsing error occurred when the "previewFeatures = ["interactiveTransactions"]" is added in shema.prisma

### DISCUSSION THREAD

**Discussion by leesj020925** [prisma/prisma#11894] [Q&A]
I'm using mysql and while working on project with prisma, some of works are required to use "transactions".
So i added the previewFeatures in the schema.prisma file to use interactiveTransactions, however, it caused error like below when i execute "npm run migrate"
```
Error: Schema parsing
error: The preview feature "interactiveTransactions" is not known. Expected one of: microsoftSqlServer, orderByRelation, nApi, selectRelationCount, orderByAggregateGroup, filterJson, planetScaleMode, 
referentialActions
  -->  schema.prisma:12
   | 
11 |   provider        = "prisma-client-js"
12 |   previewFeatures = ["interactiveTransactions"]
   | 

Validation Error Count: 1
```
I couldn't find any references about this so i open an Q&A here. Thanks!

**Comment by janpio** [comment]
What version of Prisma CLI are you using?

**Reply by leesj020925** [reply]
the version is 2.30.3. is it too low version for that?

**Reply by janpio** [reply]
In theory not, `interactiveTransactions` was added with 2.29.0: https://github.com/prisma/prisma/issues/8664 But the error message indicates your CLI does not know about it. Are you sure the CLI you are running to migrate is really 2.30.3?

**Reply by vladi-stevanovic** [reply]
👋 Hello @leesj020925  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
